packages/tests/src/api/users/oauth.ts

```diff
@@ -79,14 +81,17 @@ describe('Test oauth', function () {
     })
 
     it('Should not login with an invalid password', async function () {
-      const user = { username: server.store.user.username, password: 'mew_three' }
+      const user = { username: 'User@example.com', password: 'password' }
```

Comment:
Please duplicate the test instead of replacing it (you can for example use multiple login in this test)